app/assets/javascripts/feed_entries.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
+  var fetchRole = "a[data-role='fetch']";
+  $(document).on("ajax:success", fetchRole, function (e, data, status, xhr) {
+    console.log("Fetching feed");
+    $("div#flash").replaceWith(xhr.responseText);
+  })
+  .on("ajax:error", fetchRole, function(e, xhr, status, error) {
+    console.log("Error");
+  })
+  .on("ajax:complete", fetchRole, function(e, xhr, status) {
```

Comment:
'status' is defined but never used.<br>'xhr' is defined but never used.<br>'e' is defined but never used.